### ISSUE TITLE
Bookmarked songs from BeastSaber are not hashed

### ISSUE THREAD

**Issue by DangerousDetlef** [Zingabopp/BeatSync#46]
Some bookmarked songs from BeastSaber are not getting hashed, insted they are skipped:

![2021-09-02_17h31_34](https://user-images.githubusercontent.com/5159043/131873853-7a7fbe46-e9a8-487f-9641-9b5f5a80e3b3.png)

Some others seem to work, but I can't figure out the difference.

These maps are not in the custom maps folder in the game.
Unbookmarking them, running BeatSyncConsole, bookmarking them again, running Console again changes nothing.
Removed my config folder, let Console create a new one, added only my BeastSaber Username and the Game Path, ran again, same problem.

Running latest release: BeatSyncConsole v1.0.0-Beta1 (but had the problem with at least 2 release before that)

Example for a song not working: https://bsaber.com/songs/7a64/

I really like the Console and I will try out the ingame mod, too, but it would be super great if you could fix that so that I can be sure that my favorite maps are there when I play.

Thanks for all your effort to make this game awesome!

**Comment by Zingabopp** [comment]
Delete the `BeatSyncHistory.json` file in your Beat Saber `UserData` folder and run it again (show me the log from that run). I swear I removed it skipping songs that have previously 404'd (in theory not trying to download songs that have been deleted from Beat Saber), but maybe I missed something.

**Comment by DangerousDetlef** [comment]
Deleting the `BeatSyncHistory.json` did the trick. Although I'm not sure how it happened, since those songs were not ones I had added before - maybe they just 404'ed the first time because bsaber was down (or at least the API)? Anyways, that resolved my issue with these songs and if it happens again I know what to do. I attached my log file of that run, though. Thanks for your quick response and thanks again for your effort, you make having my favorite custom maps in Beat Saber really easy :)
[log.txt](https://github.com/Zingabopp/BeatSync/files/7100729/log.txt)
